user-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/user/kafka/KafkaContractTest.java

```diff
@@ -230,7 +169,7 @@ void testRequestSet() {
                                 assertThat(result.headers().lastHeader(Token.HEADER).value())
                                         .isEqualTo(ContractTestData.REQUEST_TOKEN.getBytes())),
                 () -> assertThat(results.get(0).value().get("user").get("userId").get("id").asText())
-                        .isEqualTo(ContractTestData.USER_ID.getId()),
+                        .isEqualTo("test-user-id"),
```

Comment:
Suggest reverting to use ContractTestData

Reply:
Reverted